src/missing-types.d.ts

```diff
@@ -312,10 +315,11 @@ declare module '@higlass/utils' {
     import type { ScaleContinuousNumeric } from 'd3-scale';
     import type { TilesetInfo } from '@higlass/services';
 
-    type ChromInfo<Name extends string> = {
+    export type ChromInfo<Name extends string = string> = {
```

Comment:
Just learned the `= string` syntax :)

Could you teach me why `= string` is necessary when `Name` is extending `string`? I assume this is because the `chrom` of `chrToAbs` allows non-string types (e.g., `chr1` | `chr1` | ...)?

Reply:
`= string` provides a default for the generic so you can use the type without `<>` if desired. I think it's sensible to have a default because we likely don't have enough type information often (so falling back to `string` is most flexible). But in the case where we do have more information (the chrom names), we can capture that.

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgMIAsoHsC2BJEGLAdWDHSwFcwARCeSgGzAB4A5OHFCAD0hAAmAZ2RCwUUAHMAfMgDeAKGTIElHAAUsQssCwghALnnIADlqMg1AI2gBuFZiNiJIScgC+AbQC6tpQ6hNbTBdfSMAJQgELCgBdk4IABpjM0NkSxwbKA9pP2UETFwAGQhXcjTI6Nj4rmSMrNz-MCwwOEYSsvQLazsFdwUFASjGOCgUaP0wZEgxAAYjDGx8QhIyCmo6BmY-QeHR8b0xaYgxAEYFwuWiUnIqWno4JlZnKUbFfMOpuVMtD2QAXmOZwAdAVAlodIdgVYRuhkAB6eHpLDIaDYKAAQj6CneKk+KV+7gBQLAp1BmCCkP05Kgp2xuwQIzGeMmJIATBclgRrms7ptHswWAByMGnIVvfwTI7fVJ-QEzMBsmmUkJQmFwdDY3FSr4-ERE+UnRXKiGq6mi+kKUCQWCIFCLXDcrA1bh8UrCUTiKTEl6uWTatQq0JpGXmdI9KD2MFOL2uDw+PIBIOHCJRGJxDi1AlperQHKJsHFUqScqpqoZhJ1CONZTNVrtYvkbqZXr9BlMg6s0BETmOlb2RGiBLIOAiXicEyMFBskdWLAANwgA21+ND+uJ3awJuCwehsK1ktXerlyE326pQhpdLbQ0Z+xZR035zQlydwtF4r8K9Za5PT-PZqXuqmr9N+0rHgap4rGSYLJuamDXgMt4dsgMCUCACBmgEAAqWAAIJWEILqom6ggiL6MgABQgAkRiZkkUE9i+XIrC60gAJTNlkOxgrhBFCJRQroBAjCMFgQrJJu7EKLx+GEYJwmieJknQexQA

Reply:
```typescript

declare const info: ChromInfo;

{
  const { pos } = info.chrPositions.blah // ok because any string key is valid
}

{
  const { pos } = info.chrPositions.chr1
}
```


```typescript

declare const info: ChromInfo<'chr1' | 'chr2'>;

{
  const { pos } = info.chrPositions.blah // error!!! not 'chr1' or 'chr2' 
}
```